core/project/WhitesourceLicensePlugin.scala

```diff
@@ -1,16 +1,33 @@
 import sbt._
-
+import sbt.Keys._
 import sbtwhitesource.WhiteSourcePlugin.autoImport._
+import sbtwhitesource._
+import scala.sys.process.Process
+import scala.util.Try
+
+object Whitesource extends AutoPlugin {
+  private lazy val gitCurrentBranch =
+    Try(Process("git rev-parse --abbrev-ref HEAD").!!.replaceAll("\\s", "")).recover {
+      case e => sys.error(s"Couldn't determine git branch for Whitesource: $e")
+    }.toOption
+
+  override def requires = WhiteSourcePlugin
+
+  override def trigger = allRequirements
 
-object LicensePlugin extends AutoPlugin {
-  override def requires = plugins.JvmPlugin
-  override def trigger  = allRequirements
+  override lazy val projectSettings = Seq(
+    // do not change the value of whitesourceProduct
+    whitesourceProduct := "cloudflow",
+    whitesourceAggregateProjectName := {
+      "cloudflow-2.0-" + (
```

Comment:
```suggestion
      "cloudflow-" + (
```
The version should come from `version` (as it does for non-snapshots below) and not be part of the `master` and `adhoc` updates.

Reply:
updated! Need to wait for next RC release before merging (already triggered)